gramps/gen/lib/test/datadict_test.py

```diff
@@ -0,0 +1,121 @@
+#
+# Gramps - a GTK+/GNOME based genealogy program
+#
+# Copyright (C) 2025      Doug Blank
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+
+""" Unittest for DataDict """
+
+# See also tests in ./serialize_test.py
+
+import unittest
+
+from gramps.gen.lib.serialize import DataDict, DataList, to_dict, from_dict
+from gramps.gen.lib import (
+    Person,
+    Family,
+)
+
+
+class DataDictTest(unittest.TestCase):
+    def test_person(self):
+        p = Person()
+        d = DataDict(p)
+        # Get the object from the dict:
+        assert id(p) == id(from_dict(d))
```

Comment:
Use `self.assertEqual` instead.

Reply:
Sure, I can make those self.assert changes. Do you know about the "Add a suggestion" button when making a comment?

Reply:
No.  I haven't used that before.

Reply:
Here is a short description with screen shot: https://haacked.com/archive/2019/06/03/suggested-changes/

Reply:
You can do multi-line suggestions as well.
One gotcha is that the suggestion may fall foul of our linting rules. This happened elsewhere in this PR where I suggested a longer message which then failed linting. Doug had to go back and make a new commit to correct. Sorry @dsblank